File: Php/src/joseahernandez/fizzbuzz/FizzBuzz.php
<?php

namespace joseahernandez\fizzbuzz;

class FizzBuzz
{
    public function fizzBuzz($number)
    {
        $result = '';
        if ($this->isFizz($number))
            $result = 'Fizz';
        if ($this->isBuzz($number))
            $result .= 'Buzz';

        if (!$this->isFizz($number) && !$this->isBuzz($number))
            $result = (string)$number;

        return $result;
    }

    private function isFizz($number)
    {
        return $number % 3 == 0;
    }

    private function isBuzz($number)
    {
        return $number % 5 == 0;
    }
}

File: Kotlin/src/main/kotlin/joseahernandez/fizzbuzz/FizzBuzz.kt
package joseahernandez.fizzbuzz

class FizzBuzz {
    fun fizzBuzz(number: Int): String {
        var result = ""
        if (isFizz(number)) result = "Fizz"
        if (isBuzz(number)) result += "Buzz"

        if (!isFizz(number) && !isBuzz(number)) result = number.toString()

        return result
    }

    private fun isFizz(number: Int) = number % 3 == 0

    private fun isBuzz(number: Int) = number % 5 == 0
}

fun main(args: Array<String>) {
    val fizzBuzz = FizzBuzz()

    for ( i in 1..100) {
        println(fizzBuzz.fizzBuzz(i))
    }
}